### ISSUE TITLE
testParseTimestampTzLCR fails when TZ other than Pacific/Auckland

### ISSUE THREAD

**Issue by h1f1x** [dbvisitsoftware/replicate-connector-library#1]
just got an error, when executing try to build the project:
mvn clean install

```
...
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 27.124 sec - in com.dbvisit.replicate.plog.file.PlogFileManagerTest

Results :

Failed tests:
  LogicalChangeParserTest.testParseTimestampTzLCR:322->testParseInsertLCR:589 Expecting column id: 1 name: TEST value: 2016-09-01 18:00:01.0, got: Column 1 name: TEST type: TIMESTAMP_WITH_TIME_ZONE value: 2016-08-31 12:00:01.0

Tests run: 81, Failures: 1, Errors: 0, Skipped: 0
...
```

If I changed my local timezone to:
Pacific/Auckland

it worked.
To get this error: simply do:
```
TZ='Europe/Berlin'; export TZ
mvn clean install
```

**Comment by dizaknz** [comment]
This is a known issue that will be resolved in the next snapshot release 2.9.00 of the replicate connector library. Thank you.

**Comment by dizaknz** [comment]
Issue resolved in 2.9.00
